### ISSUE TITLE
[feature request] Hide standard tab bar upon showing TST and vice versa

### ISSUE THREAD

**Issue by Mehradzie** [piroor/treestyletab#1511]
Hide standard tab bar upon showing TST and vice versa

When pressing `F1` TST would show and the standard tab bar hides. Press `F1` again and the standard tabs show back up and TST hides as there is no use for both of them to be on the same screen at the same time.

 * Platform (OS): Windows 7
 * Version of Firefox: 57.0b14
 * Version (or revision) of Tree Style Tab: 2.0.7

**Comment by kah0922** [comment]
That can't happen until [Bug 1332447](https://bugzilla.mozilla.org/show_bug.cgi?id=1332447) lands.

**Comment by rancidfrog** [comment]
Yes it can. This can be easily fixed temporarily by having a Preference option that creates `userChrome.css` with:
```
#TabsToolbar {
    visibility: collapse !important;
}
```
Actually, some people can argue that this should be done automatically as the whole point of the add-on is to replace tabstoolbar with side tabs toolbar.

**Comment by gnite** [comment]
No it can't. The only way to create and manage the userChrome.css file is to do it manually. Add-ons can't do that.

**Comment by me10240** [comment]
Could anyone explain where should the file userChrome.css be placed ? What is the process to use the workaround mentioned ?

**Comment by piroor** [comment]
There is a document about userChrome.css on MDN:
https://developer.mozilla.org/en-US/docs/Mozilla/Tech/XUL/Tutorial/Modifying_the_Default_Skin#Customize_with_userChrome.css

**Comment by piroor** [comment]
And you'll find the "user profile directory" from "about:support".

**Comment by evactadmin** [comment]
I just had this issue after my Firefox updated to Quantum and was able to solve it by following the above directions, but it took me a while to figure out what to do. For the benefit of others in my situation, here are the steps I followed:

1. Search Windows for %APPDATA%\Mozilla\Firefox\Profiles\ and find your user directory. In my case, I had two directories in Profiles and I opened the one with a date showing that it had been modified today.

2. Inside your user directory, create a new directory called 'chrome'. In that directory, create a new file called userChrome.css and edit it to include this:

`#TabsToolbar {
    visibility: collapse !important;
}`

3. Close Firefox and reopen it.

That worked for me.

Thanks, piroor, for the great extension!
